tests/hipscat_import/verification/test_verification_arguments.py

```diff
@@ -0,0 +1,65 @@
+"""Tests of argument validation"""
+
+
+import pytest
+
+from hipscat_import.verification.arguments import VerificationArguments
+
+
+def test_none():
+    """No arguments provided. Should error for required args."""
+    with pytest.raises(ValueError):
+        VerificationArguments()
+
+
+def test_empty_required(tmp_path, small_sky_object_catalog):
+    """*Most* required arguments are provided."""
+    ## Input path is missing
+    with pytest.raises(ValueError, match="input_catalog_path"):
+        VerificationArguments(
+            output_path=tmp_path,
+            output_catalog_name="small_sky_object_verification_report",
+        )
+
+
+def test_invalid_paths(tmp_path, small_sky_object_catalog):
+    """Required arguments are provided, but paths aren't found."""
+    ## Prove that it works with required args
+    VerificationArguments(
+        input_catalog_path=small_sky_object_catalog,
+        output_path=tmp_path,
+        output_catalog_name="small_sky_object_verification_report",
+    )
```

Comment:
The `small_sky_object_catalog` referenced here is a pytest fixture, and injected at runtime. It would be cute if we could do something like:

```
def test_something(specific_use=generic_fixture_value):
  ...
```

Though I guess an alternative convention might be:

```
def test_something(generic_fixture_value):
  specific_use = generic_fixture_value
  ...
```